packages/compass-aggregations/src/plugin-name.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export function AggregationsPluginName() {
```

Comment:
This is more a AggregationsTabTitle, not a PluginName? Just a bit weird of a naming for this component as it's being used in a pretty specific context

Reply:
makes sense, changed in [88df68d](https://github.com/mongodb-js/compass/pull/6267/commits/88df68d3ef43281b912cbfc6575abd314bf0854f)